kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -13868,5 +13869,16 @@ private String getMacroValue(String macroName) {
 		return macroName;
 	}
 
+	private String getDisplayName(User userIn,boolean isUserdir) {
```

Comment:
`isUserdir` seems redundant as can't you always pass in the user you are want to build the display name for?